genshi/template/tests/eval.py

```diff
@@ -480,6 +480,27 @@ def __repr__(self):
             self.assertEqual('index.html', code.co_filename)
             self.assertEqual(50, frame.tb_lineno)
 
+    def test_getitem_with_constant_string(self):
+        data = dict(dict={'some': 'thing'})
+        Expression("dict['some']").evaluate(data)
```

Comment:
We should assert something about what the result of `.evaluate(data)` returns in these test.

Reply:
You mean something like 632a0483c49ff9547960324c09ff6093d2ab91de ?

Reply:
Yep! Perfect.